kuma/static/styles/minimalist/document.scss

```diff
@@ -23,7 +23,7 @@ Utitlity classes (reused in a variety of contexts)
 /*
 Components that may appear on most wiki pages
 ====================================================================== */
-@import '../components/wiki/structure';
+@import 'base/structure';
```

Comment:
Would it be safer and saner to use `./base/structure` here? So it's always relative?